### PR TITLE
Set up the blocks to be translated, and add a POT file

### PR DESCRIPTION
**Pull request by laurelfulford**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR starts the process of translating the Newspack Blocks.

In it I:

* Added the text-domain `newspack-blocks`, to the JS files (the text-domain that the blocks orginially used, `newspack`, didn't work for the JS, so I:
* Updated the text domain in the .php files from `newspack` to `newspack-blocks`
* Unfortunately included formatting changes that came from my Prettier plugin; if this makes it a pain to review, I can redo the changes on fresh files without the formatting changes.

See #74 

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build:webpack`
2. Confirm that the text-domain changes throughout the JS and PHP are correct.
3. Make sure the POT file appears to include strings from JS and PHP files.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### CODE REVIEW

File: src/blocks/donate/edit.js [hunk 1]
```diff
@@ -255,10 +295,15 @@ class Edit extends Component {
...
 				<InspectorControls>
-					<PanelBody title={ __( 'Donate Block' ) }>
-						<p>{ __( 'The Donate Block allows you to collect donations from readers. The fields are automatically defined based on your donation settings.' ) }</p>
-						<ExternalLink href='/wp-admin/admin.php?page=newspack-donations-wizard#/'>
-							{ __( 'Edit donation settings.' ) }
+					<PanelBody title={ __( 'Donate Block', 'newspack-blocks' ) }>
+						<p>
+							{ __(
+								'The Donate Block allows you to collect donations from readers. The fields are automatically defined based on your donation settings.',
+								'newspack'
```
Comment (jeffersonrabb) [inline src/blocks/donate/edit.js]: Should this be `newspack-blocks`?

```suggestion
								'newspack-blocks'
```

File: src/blocks/donate/edit.js [hunk 2]
```diff
@@ -233,12 +270,15 @@ class Edit extends Component {
...
 			return (
 				<Placeholder
-					icon='warning'
-					label={ __( 'Not ready' ) }
-					instructions={ __( 'You have not set up your donation settings yet. You need to do that before you can use the Donate Block.' ) }
+					icon="warning"
+					label={ __( 'Not ready', 'newspack-blocks' ) }
+					instructions={ __(
+						'You have not set up your donation settings yet. You need to do that before you can use the Donate Block.',
+						'newspack'
```
Comment (jeffersonrabb) [inline src/blocks/donate/edit.js]:
```suggestion
						'newspack-blocks'
```

File: src/blocks/donate/edit.js (hunk 1 above)
Comment (laurelfulford) [reply src/blocks/donate/edit.js]: D'oh, thanks Jeff! Fixed in ae290c3.

File: src/blocks/donate/edit.js (hunk 2 above)
Comment (laurelfulford) [reply src/blocks/donate/edit.js]: Thanks for catching that - fixed in ae290c3!

**Comment by laurelfulford** [comment]
Thanks Jeff! I fixed those in ae290c3, and regenerated the POT file, so it added those two strings.

**Comment by laurelfulford** [comment]
Thanks!

**Pull request merged**